build.sbt

```diff
@@ -0,0 +1,47 @@
+import com.typesafe.sbt.SbtGit._
+
+name := "lenthall"
+
+version := "0.13-SNAPSHOT"
+
+organization := "org.broadinstitute"
+
+scalaVersion := "2.11.7"
+
+libraryDependencies ++= Seq(
+  "com.typesafe" % "config" % "1.2.1",
```

Comment:
Considered part of the common code. Anything not `Provided` will become a transitive dependency, and will not need to be included.